src/modules/ApiSimpson.js

```diff
@@ -0,0 +1,20 @@
+import { API_URL1 } from './Vars.js'; // , API_URL2
+
+// async function getData() {
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
well resolved! ✔️